R/rename.R

```diff
@@ -61,5 +60,10 @@ rename.hyperSpec <- function(.data, ...){
     expect_equal(group_vars(rename_grouped_df), "a_newcolname")
   })
 
+  # UT4
+  test_that("renaming spectrum column throws error", {
+    expect_error(chk.hy(rename(chondro, spc_newcolname = spc)))
```

Comment:
`chk.hy()` is not needed here - should be done inside `rename.hyperSpec` since otherwise an invalid hyperSpec object may be returned.